src/common/Button/Button.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import './button.scss'
+
+const Button = ({ classList, disabled, label, onClick, type }) => {
+  const buttonClassName = classNames('btn', `btn-${type}`, classList)
+
+  return (
+    <button
+      className={buttonClassName}
+      disabled={disabled}
+      onClick={onClick}
+      type="button"
+    >
+      {label}
+    </button>
+  )
+}
+
+Button.defaultProps = {
+  classList: '',
+  disabled: false,
+  label: 'Button',
+  onClick: () => {}
+}
+
+Button.propTypes = {
+  classList: PropTypes.string,
+  disabled: PropTypes.bool,
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+  onClick: PropTypes.func,
+  type: PropTypes.string.isRequired
```

Comment:
1. Let‘s rename `type` to `variant` (I borrowed that name from some UI component library).
2. Since the `Button` component does not expect just _any_ string value here, let‘s limit it to a close list of the available values, by using `PropTypes.oneOf(…)` instead of `PropTypes.string`. This will also make StoryBook display a nice radio button field to select one of the listed options.
  ![image](https://user-images.githubusercontent.com/13918850/110638633-76f99b00-81b7-11eb-83ae-513c6eb952d6.png)


```suggestion
  variant: PropTypes.oneOf([
    'primary',
    'secondary',
    'tertiary',
    'danger',
    'label'
  ]).isRequired
```